content/events/17-18/delaware-talk.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Delaware TechZ talk'
+description: Two experts discuss how .
+created_at: 10-04-2018
+time: '18-04-2018 19:30'
+end: '18-04-2018 22:00'
+location: 'Auditorium D, Jozef Plateaustraat 22, 9000 Gent'
+locationlink: 'Faculteit Ingenieurswetenschappen en Architectuur'
+facebook: 'https://www.facebook.com/events//'
+color: '#aea485'
+gradient: false
+#image: 'https://zeus.ugent.be/zeuswpi/Ow2Lrgdx.svg'
+---
+
+This event is part of the [TeCH TalkZ](<%= @items['/blog/17-18/tech-talkz.md'].path %>), a collaboration between the two student associations Zeus WPI and CenEka.
+
+# Intro
+
+[delaware](https://www.delaware.pro/en-be), the global consultancy firm with which we co-hosted the [Lasershoot](<%= @item['/blog/17-18/lasershoot-blogpost.md].path %>) event is back. This time they will do a two-part presentation about technology and innovation at delaware. The total presenting time will be 1h30, divided among both topics. Afterwards we'd like to invite you, as per usual at talks, to a brief reception.
+
+# Part 1: Continuous web app deployment with zero downtime
+
+Did you know that it takes Google, Microsoft and Facebook only a few days or weeks to release a new version of their applications?
+During this session we'll discuss how we can use Scrum as a process to enforce continuous delivery, 
+and what it takes to deploy new stable versions of web applications with confidence (and without any downtime).
+This part will be presented by Emanuel Dejonghe, senior full-stack developer for delaware and UGhent alumnus.
```

Comment:
I don't know if UGhent is a 'correct' abbreviation. I would use UGent to be sure.

Reply:
Rien is partially right, from the official style guide:
> We don’t use the abbreviation ‘UGent’ in English. The correct terminology is 'Ghent University'.